src/relay.js

```diff
@@ -189,6 +181,8 @@ export function sequelizeConnection({
   let argsToWhere = function (args) {
     let result = {};
 
+    if (where === undefined) return result;
+
     _.each(args, (value, key) => {
       if (key in $connectionArgs) return;
```

Comment:
args always contains orderBy but now it's not necessarily inside $connectionArgs so orderBy can end up inside the WHERE clause: #570
Should we force a check here for key equaling `orderBy` or is there a fix to be applied elsewhere?

Reply:
We currently set `args.orderBy`, that should be rewritten, it's not exactly side effect free or obvious unless you look at the source